packages/fxa-auth-server/lib/profile/client.js

```diff
@@ -4,73 +4,76 @@
 
 'use strict';
 
-const isA = require('joi');
-const createBackendServiceAPI = require('../backendService');
+import axios from 'axios';
 
 const PATH_PREFIX = '/v1';
 
-// Very generic validator, because there's not really a useful response
-// here other than that it didn't fail in error
-const DeleteCacheResponse = isA.any();
+// Why is this subhub service vs "profile"?
+const serviceName = 'subhub';
```

Comment:
Surely that was a bug??  I can't think of a reason why we would want to use 'subhub' here.